src/net/sourceforge/kolmafia/session/ResponseTextParser.java

```diff
@@ -146,44 +146,75 @@ public static boolean hasResult(final String location) {
       queryString = location.substring(queryStringBegin + 1);
     }
 
+    // *** This is an unfinished and unused feature.
     if (!PageRegistry.isGameAction(path, queryString)) {
       return false;
     }
 
-    if (location.startsWith("lchat.php")
-        || location.startsWith("mchat.php")
-        || location.startsWith("newchatmessages.php")
-        || location.startsWith("submitnewchat.php")) {
-      return false;
-    }
-
-    if (location.startsWith("showplayer.php")) {
-      // We want to register new items from Equipment and Familiar
-      return true;
-    }
-
-    if (location.startsWith("displaycollection.php")) {
-      // We want to register new items in display cases
-      return true;
+    switch (path) {
+      case "login.php", "logout.php" -> {
+        return false;
+      }
+      case "api.php", "game.php", "doc.php" -> {
+        return false;
+      }
+      case "topmenu.php", "awesomemenu.php" -> {
+        return false;
+      }
+      case "actionbar.php" -> {
+        return false;
+      }
+      case "lchat.php", "mchat.php", "newchatmessages.php", "submitnewchat.php" -> {
+        // Nothing from chat.
+        return false;
+      }
+      case "desc_effect.php",
+          "desc_familiar.php",
+          "desc_guardian.php",
+          "desc_item.php",
+          "desc_monster.php",
+          "desc_outfit.php",
+          "desc_skill.php" -> {
+        // No object descriptions
+        return false;
+      }
+      case "showplayer.php" -> {
+        // We want to register new items from Equipment and Familiar
+        return true;
+      }
+      case "displaycollection.php" -> {
+        // We want to register new items in display cases
+        return true;
+      }
+      case "managecollectionshelves.php" -> {
+        return false;
+      }
+      case "clan_stash.php",
+          "clan_hall.php",
+          "clan_rumpus.php",
+          "clan_viplounge.php",
+          "clan_hobopolis.php",
+          "clan_slimetube.php",
+          "clan_dreadsylvania.php" -> {
+        // Other clan*.php URLs have no result.
+        // Which are those?
+        return true;
+      }
+      case "questlog.php" -> {
+        return false;
+      }
     }
 
-    if (location.endsWith("menu.php") || location.startsWith("actionbar")) {
```

Comment:
these are potentially unmapped completely:
* ending with `menu.php`
* starting with `actionbar`
* starting with `desc`
* starting with `quest`

It's probably fine.

Reply:
I added specific pages, complete with .php, in the switch statement.
So, rather then "*menu.php", we have "topmenu.php" and "awesomemenu.php".
Which is to say, this is intentional.